ui/ui-components/pages/app/[id]/refresh.tsx

```diff
@@ -109,7 +109,7 @@ export default function Page(props) {
       try {
         const response: Actions.AppRefreshQueryRun = await execApi(
           "post",
-          `/appRefreshQuery/${appRefreshQuery.id}/query`
+          `/appRefreshQuery/${appRefreshQuery.id}/run`
```

Comment:
I wonder if in the next version of Actionhero, actions 'claim' their routes. If we did that, then we could get these paths dynamically..